Description: Hide Help button (until GNOME provides the help for 3.x again)
Author: Michael Terry <michael.terry@canonical.com>
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=658712
Bug-Ubuntu: https://launchpad.net/bugs/838539

Index: gnome-session-3.4.2.1/capplet/gsm-properties-dialog.c
===================================================================
--- gnome-session-3.4.2.1.orig/capplet/gsm-properties-dialog.c	2011-09-26 16:55:25.000000000 +0200
+++ gnome-session-3.4.2.1/capplet/gsm-properties-dialog.c	2012-09-05 00:39:59.797259138 +0200
@@ -474,7 +474,6 @@
         GtkTargetList     *targetlist;
 
         gtk_dialog_add_buttons (GTK_DIALOG (dialog),
-                                GTK_STOCK_HELP, GTK_RESPONSE_HELP,
                                 GTK_STOCK_CLOSE, GTK_RESPONSE_CLOSE,
                                 NULL);
 
